content-src/components/DiscoveryStreamComponents/List/_List.scss

```diff
@@ -206,7 +206,10 @@ $item-line-height: 20;
 
   .ds-list-item-excerpt {
     @include limit-visibile-lines(2, $item-line-height, $item-font-size);
-    color: $grey-10-80;
+    @include dark-theme-only {
+      color: $grey-10-80;
+    }
+    color: var(--newtab-text-secondary-color);
```

Comment:
I've been avoiding using browser vars for DS and using explicit vars from `content-src/styles/_variables.scss` instead for clarity (like you did for the `dark-theme-only` case above).

Can you do that so it's consistent with the rest of the DS styles?

Reply:
That's fine with me.